contrib/babelfishpg_common/src/babelfishpg_common.h

```diff
@@ -27,6 +27,13 @@
 #define BBF_Pragma_IgnoreFloatConversionWarning_Pop \
     _Pragma("GCC diagnostic pop")
 
+typedef enum TdsAtAtVarType 
+{
+  rcount_type, 
```

Comment:
Values should be in capital letter.

Reply:
Will do. Thanks

Reply:
Done. Sorry for missing it in the first pass.